src/hooks/http.ts

```diff
@@ -346,7 +346,9 @@ export class Http {
         if (isEmptyString(requestData.body)) {
           const body = extractBodyFromEmitSocketEvent(args[1]);
           if (body) {
-            requestData.body += body.substr(0, getEventEntitySize(false));
+            const eventEntitySize = getEventEntitySize(false);
+            requestData.body += body.substr(0, eventEntitySize);
+            requestData.truncated = eventEntitySize < requestData.body.length;
```

Comment:
Although I do agree that this code should be changed to:

```suggestion
            const newBody = requestData.body + body;
            requestData.body = newBody.substr(0, eventEntitySize);
            requestData.truncated = eventEntitySize < newBody.length;
```

Reply:
please add tests to these two cases